app/models/service_ansible_playbook.rb

```diff
@@ -14,6 +14,25 @@ def preprocess(action, add_options = {})
   end
 
   def execute(action)
+    task_opts = {
+      :action => "Executing Ansible Playbook",
+      :userid => "system"
```

Comment:
Should this be the current user who ordered the service or the service owner?

Reply:
Maybe? I wasn't sure if this was relevant as the user didn't seem to be used, but I can change it. How to I get the userid for the user who ordered the service (I would assume that's the better option of the two)?

Reply:
I think this should actually stay as system, because it's just an internal placement mechanism, not something we need user context for.  I think the "Executing Ansible Playbook" action name is what makes that confusing and we should change that to "Launching Ansible Job" or something.